perfectscale/README.md

```diff
@@ -0,0 +1,43 @@
+# Agent Check: Perfectscale
+
+## Overview
+
+
+PerfectScale is a comprehensive cloud-agnostic solution that enables teams with continuous and autonomous Kubernetes optimization capabilities. Our main goal is to empower teams to effortlessly manage their infrastructure, make quick, data-driven decisions, and reach peak performance at the lowest possible cost.
+
+PerfectScale monitors resiliency metrics across your Kubernetes environment and sends these metrics to Datadog through the Integration. When PerfectScale identifies under-provisioned or misconfigured workloads, it promptly pushes an alert to the Datadog dashboard detailing the issue type (such as OOM, CPU-Throttling, Eviction, and more) along with its precise location (cluster, namespace, workload, and container).
+
+## Setup
+
+### Create account
+
+If you don't have a PerfectScale account yet, sign up for a [free 30-day trial][1]. Once it is active, you can start using the PerfectScale integration.
+
+### Configuration
+
+1. [Sign in to][2] PerfectScale.
+2. To start using the Datadog integration, the cluster should be connected to PerfectScale. If this is not completed yet, add the cluster by following the instructions in the [official PerfectScale documentation][3].
+![Perfectscale Screenshot][4]
+3. Create an [API Key][5] in Datadog for the PerfectScale integration.
+4. [Create a Datadog profile][6].
+5. Paste your API Key and select the associated [site][7].
+6. Apply the profile to the desired cluster(s).
```

Comment:
```suggestion
1. [Sign in to][2] PerfectScale.
2. To start using the Datadog integration, the cluster should be connected to PerfectScale. If this is not completed yet, add the cluster by following the instructions in the [official PerfectScale documentation][3].
![Perfectscale Screenshot][4]

3. Create an [API Key][5] in Datadog for the PerfectScale integration.
4. [Create a Datadog profile][6].
5. Paste your API Key and select the associated [site][7].
6. Apply the profile to the desired cluster(s).
```

Reply:
When looking in our stagin env, the steps below the image are in the same line. Can you try adding this space to see if this resolves the issue?

Reply:
Done